### PR TITLE
Adding devices to ConditionalAccessConditionSet

### PR DESCRIPTION
**Pull request by gapalani**

### CODE REVIEW

File: api-reference/beta/api/conditionalaccessroot-post-policies.md [hunk 1]
```diff
@@ -548,6 +548,122 @@ Content-type: application/json
...
+            "excludeUsers": [],
+            "includeGroups": [
+                "ba8e7ded-8b0f-4836-ba06-8ff1ecc5c8ba"
+            ],
+            "excludeGroups": [],
+            "includeRoles": [],
+            "excludeRoles": []
+        },
+        "devices": {
+            "includeDeviceRule": null,
```
Comment (elcarlosadrian) [inline api-reference/beta/api/conditionalaccessroot-post-policies.md:646]: Shouldn't we remove the references to deviceRule from this PR? We are not ready yet

File: api-reference/beta/resources/conditionalaccessdevices.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: "conditionalAccessDevices resource type"
+description: "Represents devices in the policy scope."
+localization_priority: Normal
+author: "gapalani"
```
Comment (elcarlosadrian) [inline api-reference/beta/resources/conditionalaccessdevices.md:5]: I think we used David Murray as the author for these files

File: concepts/changelog.md [hunk 3]
```diff
@@ -122,6 +122,7 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 | **Change type** | **Version** | **Description**                  |
 |:----------------|:------------|:-----------------------------------------|
 | Change | beta | Updated the conditional access APIs to use the /identity/ segment; for example, /beta/identity/conditionalAccess/policies.|
+| Addition | beta | Added the **devices** property to the [conditionalaccessconditionset](/graph/api/resources/conditionalaccessconditionset?view=graph-rest-1.0) entity. |
```
Comment (elcarlosadrian) [inline concepts/changelog.md:125]: is the query string parameter ?view=graph-rest-1.0 needed? Does it even exist?

File: api-reference/beta/api/conditionalaccessroot-post-policies.md (hunk 1 above)
Comment (gapalani) [reply api-reference/beta/api/conditionalaccessroot-post-policies.md]: Yes done

File: api-reference/beta/resources/conditionalaccessdevices.md (hunk 2 above)
Comment (gapalani) [reply api-reference/beta/resources/conditionalaccessdevices.md]: Done.

File: concepts/changelog.md (hunk 3 above)
Comment (gapalani) [reply concepts/changelog.md]: My bad, copy pasted it. Fixed it

**Comment by Lauragra** [comment]
@gapalani , what is the merge status of this PR?

**Comment by Lauragra** [comment]
@gapalani , this API doesn't appear to be live yet. Please set the *ready to merge* label when the APIs are publicly available. You'll need to move the changelog entry in April or whatever month it will be. Thanks.

**Pull request merged**